tools/analysis/ping-patterns/README.md

```diff
@@ -0,0 +1,71 @@
+This directory contains a tool to analyse the patterns of baseline and metrics pings received on a per-client basis in Fenix.  These pattern analyses are then summarized in a single plot showing various patterns and issues found in the data.
```

Comment:
Is this really Fenix specific? Does it really make a difference if we're able to change the table name and use it, for example, to check Firefox Reality?

*Update* oh yes, I just figured out that it's due to the build id to date thing.